tcblib/db_functions.py

```diff
@@ -0,0 +1,97 @@
+import boto3
```

Comment:
pep-8 has rules on how to do imports: https://www.python.org/dev/peps/pep-0008/#imports

1. Standard library imports.
2/ Related third party imports.
3. Local application/library specific imports.

```
import datetime
import os

import boto3
```